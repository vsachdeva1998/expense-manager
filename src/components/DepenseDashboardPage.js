import React from 'react';
import DepenseList from './DepenseList';
import DepenseListFilters from './DepenseListFilters';
import DepensesSummary from './DepensesSummary';

const DepenseDashboardPage = () => (
  <div>
    <DepensesSummary />
    <DepenseListFilters />
    <DepenseList />
  </div>
);

export default DepenseDashboardPage;
