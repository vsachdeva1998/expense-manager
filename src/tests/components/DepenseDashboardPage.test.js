import React from 'react';
import { shallow } from 'enzyme';
import DepenseDashboardPage from '../../components/DepenseDashboardPage';

test('should render DepenseDashboardPage correctly', () => {
  const wrapper = shallow(<DepenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
