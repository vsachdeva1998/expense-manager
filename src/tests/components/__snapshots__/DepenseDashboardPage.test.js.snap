// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DepenseDashboardPage correctly 1`] = `
<div>
  <Connect(DepensesSummary) />
  <Connect(DepenseListFilters) />
  <Connect(DepenseList) />
</div>
`;
